website/static/js/profile.js

```diff
@@ -676,6 +676,19 @@ var SocialViewModel = function(urls, modes, preventUnsaved) {
         return false;
     });
 
+    self.unmatchedUrls = ko.computed(function() {
+        var values = self.values();
```

Comment:
It doesn't look like values is mutated, but there is a mix-and-match of self.values() and values. Probably just pick one (I saw there was some old code that matched this style that it looks like you pulled from).